src/module/entities/helper.js

```diff
@@ -261,10 +261,19 @@ module.exports = class UserProjectsHelper {
 						message: CONSTANTS.apiResponses.ROLES_NOT_FOUND,
 					}
 				}
+				// Transforming the data
+				const transformedData = fetchUserRoles.result.map((item) => {
+					// For each item in the result array, create a new object with modified keys
+					return {
+						_id: item._id,
+						value: item.userRoleId,
+						label: item.title,
```

Comment:
@MallanagoudaB Here also you need to add roleType. After making change if you have tested and verified the response, you should have noticed it